File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.cpp

#include "SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  std::string SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow.cs
/// </filename>


namespace commands
{
    public class SelectRowCommand_ViewMyStringRowHandleTableViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
  }
}

